unity-renderer/Assets/DCLPlugins/RealmsPlugin/RealmPlugin.cs

```diff
@@ -14,18 +15,19 @@ public class RealmPlugin : IPlugin
     {
         private BaseVariable<AboutConfiguration> realmAboutConfiguration => DataStore.i.realm.playerRealmAboutConfiguration;
         private List<RealmModel> currentCatalystRealmList;
+        private IWorldsAnalytics worldAnalytics;
 
         internal List<IRealmModifier> realmsModifiers;
 
-        public RealmPlugin(DataStore dataStore)
+        public RealmPlugin(DataStore dataStore, IWorldsAnalytics analytics)
```

Comment:
What if we move `WorldsAnalytics` out of the `RealmPlugin` class, and handle its subscription to `realmAboutConfiguration.OnChange` inside the `WorldsAnalytics` itself? By doing it, we will remove one coupling between this two classes and encapsulate logic of `WorldsAnalytics` just inside it (so no public methods at all). 

1. We can register WorldsAnalytics in the Plugins as separate plugin.
2. Have subscription in the constructor (similar as for `updateEventHandler`)
3. And we will need to move `isWorld` logic inside `AboutConfiguration` (via Extension method)

What do you think about it?